### PR TITLE
android: Mark chat unread

### PR DESCRIPTION
**Pull request by avently**

### CODE REVIEW

File: apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt [hunk 1]
```diff
@@ -29,27 +29,29 @@ import kotlinx.datetime.Clock
 @Composable
 fun ChatListNavLinkView(chat: Chat, chatModel: ChatModel) {
   val showMenu = remember { mutableStateOf(false) }
-  var showMarkRead by remember { mutableStateOf(false) }
+  val showMarkRead = remember(chat.chatStats.unreadCount, chat.chatStats.unreadChat) {
+    chat.chatStats.unreadCount > 0 || chat.chatStats.unreadChat
+  }
+  val showMarkUnread = remember(chat.chatStats.unreadChat, showMarkRead) { !chat.chatStats.unreadChat && !showMarkRead }
```
Comment (epoberezkin) [inline apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt:35]: why do you need to different vars here if they are always the opposite? showMarkRead === !showMarkUnread

Comment (epoberezkin) [reply apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt:35]: I may be missing something, but I think you always show one or another button. Or is there a case when none can be shown?

File: apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt [hunk 2]
```diff
@@ -142,6 +147,9 @@ fun GroupMenuItems(chat: Chat, groupInfo: GroupInfo, chatModel: ChatModel, showM
       if (showMarkRead) {
         MarkReadChatAction(chat, chatModel, showMenu)
       }
+      if (showMarkUnread) {
```
Comment (epoberezkin) [inline apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt:150]:
```suggestion
      else {
```

File: apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt (hunk 1 above)
Comment (avently) [reply apps/android/app/src/main/java/chat/simplex/app/views/chatlist/ChatListNavLinkView.kt]: That was stupid decision, never mind. Fixed

**Pull request merged**